src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Value.txt

```diff
@@ -91,4 +91,25 @@ true
 
 >> Value(Date(1900, 3, 1)) - Value(Date(1900, 2, 28))
 /*SqlRunner*/ Compile Error
-1
\ No newline at end of file
+1
+
+>> Boolean("true")
```

Comment:
>>> [](http://example.com/codeflow?start=0&length=2)

nit: these should be in their own file, boolean.txt

Reply:
Done